src/server/stream_family_test.cc

```diff
@@ -734,4 +734,79 @@ TEST_F(StreamFamilyTest, XInfo) {
                                           ArgType(RespExpr::NIL), "lag", IntArg(0))));
 }
 
+TEST_F(StreamFamilyTest, XAutoClaim) {
+  Run({"xadd", "foo", "1-0", "k1", "v1"});
+  Run({"xadd", "foo", "1-1", "k2", "v2"});
+  Run({"xadd", "foo", "1-2", "k3", "v3"});
+  Run({"xadd", "foo", "1-3", "k4", "v4"});
+
+  // create a group for foo stream
+  Run({"xgroup", "create", "foo", "group", "0"});
+  // alice consume all the stream entries
+  Run({"xreadgroup", "group", "group", "alice", "streams", "foo", ">"});
+
+  // bob claims alice's two pending stream entries
+  // testing the mandatory command options.
+  auto resp = Run({"xautoclaim", "foo", "group", "bob", "0", "1-2"});
+  EXPECT_THAT(
+      resp,
```

Comment:
I personally tend to remove the `resp` var, and have the `Run()` statement as part of the `EXPECT`. It's shorter, and if `Run()` is a 1-liner, I think it's easier to read. Your call :)

Reply:
i see... maybe I will keep it this way to be consistent with other functions... otherwise i would change all of them...